Waste.API/Migrations/20231201092614_AddDeductionAmount.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Waste.API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddDeductionAmount : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<float>(
+                name: "DeductionAmount",
+                table: "WasteJourney",
+                type: "real",
```

Comment:
generic type is flot but the string assigned to type key is "real". Was this intentional?